modules/smart-agent_systemd-services/conf/readme.yaml

```diff
@@ -0,0 +1,5 @@
+documentations:
+  - name: Smart Agent monitor
+    url: 'https://docs.signalfx.com/en/latest/integrations/agent/monitors/collectd-systemd.html'
+
+source_doc:
```

Comment:
I would add a configuration example of the monitor here 
you check an example here : https://github.com/claranet/terraform-signalfx-detectors/tree/master/modules/smart-agent_nginx-ingress